recipe/meta.yaml

```diff
@@ -120,14 +125,22 @@ outputs:
       summary: clang compiler components in one package for bootstrapping clang
```

Comment:
Seems like the LLVM license should be `Apache-2.0 WITH LLVM-exception`, see:
- https://releases.llvm.org/14.0.0/LICENSE.TXT
- https://spdx.org/licenses/LLVM-exception.html

But is it compliant with `conda-build` and `anaconda-linter`?